docusaurus/docs/getting-started/what-s-included/jest-rtl.md

```diff
@@ -0,0 +1,8 @@
+---
+id: jest-rtl
+title: Jest & RTL
+sidebar_label: Jest & RTL
+---
+
+For testing, this boilerplate is configured to use Jest and react-testing-library, providing strong options for testing varied types of code.
```

Comment:
Maybe link to [`react-testing-library`](https://github.com/testing-library/react-testing-library) and [Jest](https://jestjs.io/)?

Reply:
Yes but first I will give a better description.

Reply:
Done, please review this section but I'm afraid I don't have enough experience with RTL to give a better description. Maybe with time..

Reply:
Review done!